integrationTests/testProcessorNode.go

```diff
@@ -541,16 +541,32 @@ func (tpn *TestProcessorNode) ProposeBlock(round uint32) (data.BodyHandler, data
 	blockHeader.SetPrevRandSeed(currHdr.GetRandSeed())
 	blockHeader.SetRandSeed(sig)
 
-	return blockBody, blockHeader
+	shardBlockBody, ok := blockBody.(dataBlock.Body)
+	txHashes := make([][]byte, 0)
+	if ok {
```

Comment:
Maybe early exit here? The code would look cleaner

Reply:
:+1: